custom-login/src/Login.jsx

```diff
@@ -46,17 +46,19 @@ const Login = () => {
         issuer,
         scopes,
       },
+      useInteractionCodeFlow: false, // Set to true, if your org is OIE enabled
     });
 
-    widget.showSignInToGetTokens({
```

Comment:
Removing the `showSignInToGetTokens` method as it will not work for MFA scenarios when NOT using interaction code flow.
Once we move completely to interaction flow mode in OIE, I guess we can bring this back.